app/uk/gov/nationalarchives/omega/editorial/services/jms/ResponseBuilder.scala

```diff
@@ -97,7 +97,7 @@ class ResponseBuilder @Inject() (stubData: StubData) {
     parse[GetAgentSummaryList](jmsMessage)
       .flatMap(agentSummaryReq =>
         asJsonString(
-          stubData.getAgentSummaries.filter(_.description.depository == agentSummaryReq.depository)
+          stubData.getAgentSummaries.filter(_.description.head.depository == agentSummaryReq.depository)
```

Comment:
The `ResponseBuilder` is a piece of code used by the `StubServer` to serve up appropriate results, so all of the data is known to the application. Filtering on depository is something we would expect the frontend to handle otherwise.